views/fe-utils/helpers.js

```diff
@@ -10,3 +10,9 @@ export const studyCountsToPercentage = (studyCount, targetTotal) => {
 
 export const formatTooltipData = (studyCounts = 0, studyTargets) =>
   !!studyTargets ? `${studyCounts} / ${studyTargets}` : `${studyCounts}`
+
+export const isTargetShown = (chartPayload) =>
```

Comment:
can we put this in a component helper since it is very specific to the `BarGraphTooltip`? Alternatively we can define it in `BarGraphTooltip.jsx` just outside of the component itself.

Reply:
https://github.com/AMP-SCZ/dpdash/pull/210/commits/663036c1a5bb8674cfe71fe25dd6ae4678e9694b